packages/ozone/tests/lang.test.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  ModeratorClient,
+  SeedClient,
+  TestNetwork,
+  basicSeed,
+} from '@atproto/dev-env'
+import AtpAgent from '@atproto/api'
+import { REASONSPAM } from '../src/lexicon/types/com/atproto/moderation/defs'
+
+describe('moderation status language tagging', () => {
+  let network: TestNetwork
+  let agent: AtpAgent
+  let sc: SeedClient
+  let modClient: ModeratorClient
+
+  beforeAll(async () => {
+    network = await TestNetwork.create({
+      dbPostgresSchema: 'ozone_blob_divert_test',
+      ozone: {
+        blobDivertUrl: `https://blob-report.com`,
+        blobDivertAdminPassword: 'test-auth-token',
+      },
+    })
+    agent = network.pds.getClient()
+    sc = network.getSeedClient()
+    modClient = network.ozone.getModClient()
+    await basicSeed(sc)
+    await network.processAll()
+  })
+
+  afterAll(async () => {
+    await network.close()
+  })
+
+  const getStatus = async (subject: string) => {
+    const { subjectStatuses } = await modClient.queryStatuses({
+      subject,
+    })
+
+    return subjectStatuses[0]
+  }
+
+  it('Adds language tag to post from text', async () => {
+    const createPostAndReport = async (text: string) => {
+      const post = await sc.post(sc.dids.carol, text)
+      // Without this delay, it seems like from time to time, the appview `getRecord` does not find the record
+      await new Promise((res) => setTimeout(res, 1000))
```

Comment:
We probably want to use `await network.processAll()` here in order to ensure all writes get indexed by the appview.

Reply:
hmm... I vaguely remember trying that but still seeing the intermittent issues but retried it just now and it works fun on re-runs! thanks.